common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationActivityFactory.java

```diff
@@ -119,6 +120,11 @@ public static Intent getAuthorizationActivityIntent(final Context context,
         final LibraryConfiguration libraryConfig = LibraryConfiguration.getInstance();
         if (ProcessUtil.isBrokerProcess(context)) {
             intent = new Intent(context, BrokerAuthorizationActivity.class);
+            if(requestUrl.contains(AuthenticationConstants.SWITCH_BROWSER.SWITCH_BROWSER)) {
```

Comment:
SWITCH_BROWSER.SWITCH_BROWSER is not very readable

Perhaps the second one should be called SWITCH_BROWSER_FLAG? or SWITCH_BROWSER_QP

Reply:
renamed